types/mapcoins.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+// map coins is a map representation of sdk.Coins
+// intended solely for use in bulk additions.
+// All serialization and iteration should be done after conversion to sdk.Coins.
+type MapCoins map[string]Int
```

Comment:
While this is clearly backed by a map, the name `MapCoins` doesn't indicate the intent of the type. Maybe `BulkCoins` or `BulkAddCoins` would be clearer.

Reply:
Sure, happy to switch to anything